archiver.go

```diff
@@ -39,6 +41,8 @@ var (
 type Archiver struct {
 	log log15.Logger
 
+	backoff *backoff.Backoff
```

Comment:
A single instance of `Archiver` can only process a single repo at a time, right?

Reply:
Right, for each job a new archiver is intantiated

Reply:
cool, then it's ok